app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -81,7 +81,7 @@ class User private constructor(
          * @param location the location of the User
          */
         fun setLocation(location: List<Double>) = apply {
-            if(location.size == SIZE_2)
+            if (location.size == 2)
```

Comment:
It was due to our previous exchange, he changed that to 2, because it was SIZE_2 which is not very meaningful either. I think we had a communication breakdown there what I meant was maybe to put something more explanatory like SIZE_OF_LOC_ATTRIBUTES etc. Because ```location.size == SIZE_2``` is not really better than ```location.size == 2```.

Reply:
yeah maybe the name could be more meaningful indeed

Reply:
At the end, I used two constants with differents names, namely `SIZE_OF_LOCATION_LIST` and `SIZE_OF_AGE_RANGE_LIST` which are both equal to 2.